pkg/apis/minio.min.io/v1/helper.go

```diff
@@ -48,9 +48,10 @@ import (
 const (
 	WebhookAPIVersion       = "/webhook/v1"
 	WebhookDefaultPort      = "4222"
-	WebhookOperatorSecret   = "operator-webhook-secret"
+	WebhookOperatorSecret   = "minio-operator-webhook-secret"
 	WebhookOperatorUsername = "webhookUsername"
 	WebhookOperatorPassword = "webhookPassword"
+	WebhookArgs             = "MINIO_ARGS"
```

Comment:
Actually this is no WebhookArgs its just `WebhookMinIOArgs`  perhaps split the constants out from here and move it to 

```suggestion
// List of supported remote ENVs for MinIO tenant deployment
const (
	WebhookMinIOArgs             = "MINIO_ARGS"
)
```